test/approvedTransfer.js

```diff
@@ -40,26 +39,24 @@ describe("Approved Transfer", function () {
   let guardianManager;
   let approvedTransfer;
   let priceProvider;
-  let kyber;
   let erc20;
   const amountToTransfer = 10000;
 
   before(async () => {
     deployer = manager.newDeployer();
     const registry = await deployer.deploy(Registry);
     const guardianStorage = await deployer.deploy(GuardianStorage);
-    kyber = await deployer.deploy(KyberNetwork);
-    priceProvider = await deployer.deploy(TokenPriceProvider, {}, kyber.contractAddress);
+    priceProvider = await deployer.deploy(TokenPriceProvider);
     await priceProvider.addManager(infrastructure.address);
     guardianManager = await deployer.deploy(GuardianManager, {}, registry.contractAddress, guardianStorage.contractAddress, 24, 12);
     approvedTransfer = await deployer.deploy(ApprovedTransfer, {}, registry.contractAddress, guardianStorage.contractAddress);
   });
+
   beforeEach(async () => {
     wallet = await deployer.deploy(Wallet);
     await wallet.init(owner.address, [approvedTransfer.contractAddress, guardianManager.contractAddress]);
     erc20 = await deployer.deploy(ERC20, {}, [infrastructure.address, wallet.contractAddress], 10000000, DECIMALS); // TOKN contract with 10M tokens (5M TOKN for wallet and 5M TOKN for account[0])
-    await kyber.addToken(erc20.contractAddress, KYBER_RATE, DECIMALS);
-    await priceProvider.syncPrice(erc20.contractAddress);
+    await priceProvider.setPrice(erc20.contractAddress, KYBER_RATE);
```

Comment:
Can we rename `KYBER_RATE` into `TOKEN_RATE` as there is no more relation to Kyber?

Reply:
Yes, makes sense